content/en/account_management/rbac/permissions.md

```diff
@@ -421,3 +422,4 @@ This permission can be granted or revoked from a role via [the Roles API][1].
 
 [1]: /account_management/users/#edit-a-user-s-roles
 [2]: /api/v2/roles/#list-permissions
+[4]: /api/v2/logs-restriction-queries/
```

Comment:
Change this to `[3]` and the corresponding reference.

Reply:
Done :)